pkg/controllers/deploy/runtime_controllers.go

```diff
@@ -43,6 +52,33 @@ func SetPrecheckFunc(checks map[string]CheckFunc) {
 	precheckFuncs = checks
 }
 
+func init() {
+	precheckFuncs := map[string]CheckFunc{
+		"alluxioruntime-controller":  alluxio.Precheck,
```

Comment:
To make it easy to understand, I suggest naming local variable `precheckFuncs` to name `precheckFuncsNoFilter`.

Reply:
Done. Renamed it to `allPrecheckFuncs`